data.sql

```diff
@@ -219,3 +219,93 @@ UPDATE animals SET owners_id = 4 WHERE name='Blossom';
 --Dean Winchester owns Angemon and Boarmon. 
```

Comment:
The project looks awesome! But the readme file could be better formatted. Please add a project title and description.
It will allow readme to look more professional.